packages/shared/src/components/cards/ActionsButtons/ActionButtons.module.css

```diff
@@ -0,0 +1,131 @@
+.upvotes {
+
+  .upvote {
+    --upvote-start-x: 0%;
+    --upvote-end-x: 50%;
+
+    --upvote-start-y: 60%;
+    --upvote-end-y: 0;
+
+    --upvote-start-rotate: 0turn;
+    --upvote-end-rotate: 0turn;
+
+    --upvote-start-scale: 1;
+    --upvote-end-scale: 1;
+
+    @apply absolute bottom-0;
+    animation: upvoteToUp 1.2s cubic-bezier(0.54, -0.59, 0.38, 0.73) forwards;
+    left: var(--upvote-start-x);
+    opacity: 0;
+    rotate: var(--upvote-start-rotate);
+    top: var(--upvote-start-y);
+    transform: translate3d(-50%, -50%, 0);
+    will-change: left, top, transform;
+
+    @media screen and (prefers-reduced-motion) {
+      animation: none;
+    }
```

Comment:
Thanks for adding this, I think it's great to have!

Just one thing I'm concerned about, is there any way we can catch that in js? 

Experiment wise, if users are enrolled but see no animation because they have the reduced motion enabled might give us fake data, which means now at experimentation phase we might either want to not add this, or add this, but also guard via js with `useConditionalFeature` where the condition would be users that do not have this enabled.